### PR TITLE
whatsnew in Celery 4.4 as per projects standard

### PR DESCRIPTION
**Pull request by auvipy**
to be completed before 14 november

### CODE REVIEW

File: docs/whatsnew-4.4.rst [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+- CPython 3.7
+- CPython 3.8
+- PyPy2.7 7.2 (``pypy2``)
+- PyPy3.5 7.1 (``pypy3``)
+- PyPy3.6 7.2 (``pypy3``)
+
+Kombu
+-----
+
+Starting from this release, the minimum required version is Kombu 4.6S.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: We'll probably release Kombu 4.6.6 and use that as a minimum version.

File: docs/whatsnew-4.4.rst [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+If you are using the MongoDB result backend please install Celery using:
+
+.. code-block:: console
+
+  $ pip install -U celery[mongodb]
+
+Redis Message Broker
+--------------------
+
+Due to multiple bugs in earlier versions of py-redis that were causing
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: We should probably mention the healthcheck changes and alert users that they should configure it.

File: docs/whatsnew-4.4.rst [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+issues for Celery, we were forced to bump the minimum required version to 3.3.0.
+
+Redis Result Backend
+--------------------
+
+Due to multiple bugs in earlier versions of py-redis that were causing
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: We should probably mention the healthcheck changes and alert users that they should configure it.

File: docs/whatsnew-4.4.rst [hunk 4]
```diff
@@ -0,0 +1,531 @@
...
+In case you are using the Riak result backend with Python 3.7, we now emit
+a warning.
+
+Please track `basho/riak-python-client#534 <https://github.com/basho/riak-python-client/issues/534>`_
+for updates.
+
+Dropped Support for RabbitMQ 2.x
+--------------------------------
+
+Starting from this release, we officially no longer support RabbitMQ 2.x.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: I think we did that last release.

File: docs/whatsnew-4.4.rst [hunk 5]
```diff
@@ -0,0 +1,531 @@
...
+The last release of 2.x was in 2012 and we had to make adjustments to
+correctly support high availability on RabbitMQ 3.x.
+
+If for some reason, you are still using RabbitMQ 2.x we encourage you to upgrade
+as soon as possible since security patches are no longer applied on RabbitMQ 2.x.
+
+Django Support
+--------------
+
+Starting from this release, the minimum required Django version is 1.11.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Have we changed that?

File: docs/whatsnew-4.4.rst [hunk 6]
```diff
@@ -0,0 +1,531 @@
...
+
+Revamped auth Serializer
+------------------------
+
+The auth serializer received a complete overhaul.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Wasn't this last release?

File: docs/whatsnew-4.4.rst [hunk 7]
```diff
@@ -0,0 +1,531 @@
...
+Kombu
+-----
+
+Starting from this release, the minimum required version is Kombu 4.6S.
+
+
+Billiard
+--------
+
+Starting from this release, the minimum required version is Billiard 3.6.1.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Has this changed?

File: docs/whatsnew-4.4.rst [hunk 8]
```diff
@@ -0,0 +1,531 @@
...
+
+Eventlet Workers Pool
+---------------------
+
+We now require `eventlet>=0.24.1`.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Copy/pasted.
Remove.

File: docs/whatsnew-4.4.rst [hunk 9]
```diff
@@ -0,0 +1,531 @@
...
+If you are using the eventlet workers pool please install Celery using:
+
+.. code-block:: console
+
+  $ pip install -U celery[eventlet]
+
+MessagePack Serializer
+----------------------
+
+We've been using the deprecated `msgpack-python` package for a while.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Copy/pasted.
Remove.

File: docs/whatsnew-4.4.rst [hunk 10]
```diff
@@ -0,0 +1,531 @@
...
+
+.. code-block:: console
+
+  $ pip uninstall msgpack-python -y
+  $ pip install -U celery[msgpack]
+
+MongoDB Result Backend
+-----------------------
+
+We now support the `DNS seedlist connection format <https://docs.mongodb.com/manual/reference/connection-string/#dns-seedlist-connection-format>`_ for the MongoDB result backend.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Copy/pasted.
Remove.

File: docs/whatsnew-4.4.rst [hunk 11]
```diff
@@ -0,0 +1,531 @@
...
+Redis Result Backend
+--------------------
+
+Due to multiple bugs in earlier versions of py-redis that were causing
+issues for Celery, we were forced to bump the minimum required version to 3.3.0.
+
+Riak Result Backend
+--------------------
+
+The official Riak client does not support Python 3.7 as of yet.
```
Comment (thedrow) [inline docs/whatsnew-4.4.rst]: Copy/pasted.
Remove.

**Comment by auvipy** [comment]
thanks for the quick review! but it's not yet ready for review :D I went trough the actual release commit history and going to add them one by on thoroughly in next 1 week before the final release. This is he higest prio formeright now :D

**Comment by thedrow** [comment]
> thanks for the quick review! but it's not yet ready for review :D I went trough the actual release commit history and going to add them one by on thoroughly in next 1 week before the final release. This is he higest prio formeright now :D

I'll help :)

**Comment by auvipy** [comment]
> > thanks for the quick review! but it's not yet ready for review :D I went trough the actual release commit history and going to add them one by on thoroughly in next 1 week before the final release. This is he higest prio formeright now :D
> 
> I'll help :)

that would be awesome. 4.4 seems appropriate! thanks!

File: docs/whatsnew-4.4.rst (hunk 1 above)
Comment (auvipy) [reply docs/whatsnew-4.4.rst]: true.

File: docs/whatsnew-4.4.rst (hunk 2 above)
Comment (adw1n) [reply docs/whatsnew-4.4.rst]: py-redis -> redis-py

**Comment by thedrow** [comment]
Actually the previous version already bumped our Django version to 1.11 if I remember correctly...

**Comment by auvipy** [comment]
you are right

**Comment by thedrow** [comment]
I'm right and you merged this anyway???? :angry:

**Comment by AndrewPix** [comment]
@thedrow Hey, as I can see from this commit https://github.com/celery/celery/commit/eb375951c72c7e47604a473bcb87b36e8cdda1a0
django version was bumped starting from 4.4.0rc1 tag

**Comment by AndrewPix** [comment]
Aww, no, you are right, it was merged to 4.3 version already

**Pull request merged**